Gemfile

```diff
@@ -25,6 +25,7 @@ group :test do
   gem "rake", ">= 10"
   gem "simplecov", ["~> 0.10", "<=0.18.2"]
   gem "concurrent-ruby", "~> 1.0"
+  gem "nokogiri", "~> 1.9"
```

Comment:
We need to run this past @tas50 - I believe we've avoided adding Nokogiri to the ecosystem as it's very difficult to support on all platforms/architectures

Reply:
Sorry this is only the `:test` group, whoops!